openwisp_firmware_upgrader/admin.py

```diff
@@ -173,7 +172,7 @@ def upgrade_selected(self, request, queryset):
                     'in this page. Refresh the page from time to time to check '
                     'its progress.'
                 )
-                self.message_user(request, mark_safe(text), messages.SUCCESS)
+                self.message_user(request, format_html(text), messages.SUCCESS)
```

Comment:
This doesn't make sense IMHO. This text is not HTML. Mark safe was doing its job fine and is more appropriate. Please revert this change. Although a minor thing, the general rule is to not change code which is working fine: if it's working don't touch it.

Reply:
I thought it would be a small improvement
But you are right it won't make any difference
I'll revert it

Reply:
@youhaveme9 always restrain from doing changes that aren't/shouldn't be part of the issue the PR is trying to solve